packages/core/test/lib/request.test.ts

```diff
@@ -0,0 +1,40 @@
+import { Event } from '@sentry/types';
+
+import { API } from '../../src/api';
+import { eventToSentryRequest } from '../../src/request';
+
+describe('eventToSentryRequest', () => {
+  const api = new API('https://dogsarebadatkeepingsecrets@squirrelchasers.ingest.sentry.io/12312012');
+  const event: Event = {
+    contexts: { trace: { trace_id: '1231201211212012', span_id: '12261980', op: 'pageload' } },
+    environment: 'dogpark',
+    event_id: '0908201304152013',
+    release: 'off.leash.park',
+    spans: [],
+    transaction: '/dogs/are/great/',
+    type: 'transaction',
+    user: { id: '1121', username: 'CharlieDog', ip_address: '11.21.20.12' },
+  };
+
+  it('adds sampling information to transaction item header', () => {
```

Comment:
Could we add one test, that tests what happens if both `__sentry_samplingMethod` and `__sentry_sampleRate` are not set?

Reply:
Done.